ale_linters/php/phpstan.vim

```diff
@@ -12,14 +12,25 @@ function! ale_linters#php#phpstan#GetCommand(buffer, version) abort
     \   ? ' -c ' . l:configuration
     \   : ''
 
+    let l:level =  ale#Var(a:buffer, 'php_phpstan_level')
+
+    if empty(l:level) && !filereadable('phpstan.neon')
```

Comment:
You should use the `ale#path#FindNearestFile` function used in other files to find this. This code will only work if the cwd for Vim contains this file.

Reply:
You are right, thanks !